partcad/src/partcad/test/cam.py

```
@@ -0,0 +1,117 @@
+from .test import Test
+from ..part import Part
+from ..part_config import PartConfiguration
+from ..assembly import Assembly
+from ..assembly_config import AssemblyConfiguration
+
+
+class CamTest(Test):
+    def __init__(self) -> None:
+        super().__init__("cam")
+
+    async def test(self, tests_to_run: list[Test], ctx, shape, test_ctx: dict = {}) -> bool:
```

Ruff (0.8.2)

Missing type annotation for function argument `ctx`

(ANN001)